packages/signals/browser-signals/src/types/signals.ts

```diff
@@ -45,10 +49,16 @@ export type InstrumentationSignal = AppSignal<
   InstrumentationData
 >
 
+interface NetworkData {
```

Comment:
Can you add the appropriate interface here 

e.g 

type NetworkRequestData = {
          action: 'Request',
          url:...
          method: 
          data: ...        
}


type NetworkResponseData = {
          action: 'Response',
          url:...
          method: 
          data: ...        
}

export type NetworkData = NetworkRequestData | NetworkResponseData